ext/crypto/import_key.rs

```diff
@@ -81,11 +82,13 @@ pub enum ImportKeyResult {
   #[serde(rename_all = "camelCase")]
   Ec { raw_data: RawKeyData },
   #[serde(rename_all = "camelCase")]
+  #[allow(dead_code)]
```

Comment:
Why is it dead code now?

Reply:
It was dead code earlier too but the warning never showed up 🤷